modules/persistent-storage-csi-snapshots-create.adoc

```diff
@@ -130,12 +138,12 @@ status:
   +
 If the value is set to `false`, the snapshot was created. However, the storage backend needs to perform additional tasks to make the snapshot usable so that it can be restored as a new volume. For example, Amazon Elastic Block Store data might be moved to a different, less expensive location, which can take several minutes.
 
-To verify that the volume snapshot was created automatically, run the following command:
-
+. To verify that the volume snapshot was created, enter the following command:
++
 ----
 $ oc get volumesnapshotcontent
 ----
++
+The pointer to the actual content is displayed. If the `boundVolumeSnapshotContentName` field is populated, there is a corresponding VolumeSnapshotContent and therefore the snapshot was created.
```

Comment:
> ! there is, there are v
Avoid at the beginning of an independent clause. In most cases rewrite to remove the phrase.

Maybe

> a VolumeSnapshotContent object exists, so the snapshot was created.